pkgs/by-name/kc/kcc/package.nix

```
@@ -0,0 +1,60 @@
+{
+  stdenv,
+  lib,
+  qt6,
+  fetchFromGitHub,
+  p7zip,
+  versionCheckHook,
+  nix-update-script,
+  python3,
+  archiveSupport ? true,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "kcc";
+  version = "7.3.3";
+
+  src = fetchFromGitHub {
+    owner = "ciromattia";
+    repo = "kcc";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-6zHUV4s1bOdARsTwNRxFM+s0p+6FLJhqJ9qG5BaBgas=";
+  };
+
+  nativeBuildInputs = [ qt6.wrapQtAppsHook ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ qt6.qtwayland ];
```

```suggestion
  buildInputs = [ qt6.qtbase ] ++ lib.optionals stdenv.hostPlatform.isLinux [ qt6.qtwayland ];
```
This is required for darwin (see ofborg)